src/kv_manager/types.rs

```diff
@@ -2,6 +2,9 @@
 
 use std::fmt::Debug;
 
+// default KV store names
+pub const DEFAULT_KV_NAME: &str = "kv";
```

Comment:
Are you changing the file path to the kv store?  If so then we may need to update docs in axelarate, etc.

Reply:
actually that is the name of the kv-store. The path remains the same at (default) "./tofnd". "kv" replaces the names of "shares" and "mnemonic" kvstores which have been merged into one and is still inside the "kvstore" directory:
```
.tofnd
└── kvstore
    └── kv
        ├── blobs
        ├── conf
        ├── db
```